examples/opendata/config

```diff
@@ -0,0 +1,53 @@
+# Pisek example task
+# See https://github.com/piskoviste/pisek/blob/master/docs/config-v3-documentation for more information
+
+[task]
+version=v3
+task_type=batch
+
+[tests]
+# Generator source file
+in_gen=gen
+gen_type=opendata-v1
+
+validator=validate
+validator_type=simple-42
+
+out_check=judge
+# Judge source file
+out_judge=judge
+judge_needs_in=1
+judge_needs_out=0
+judge_type=opendata-v2
```

Comment:
Nit: `judge_type` is rather important, perhaps it should be above `judge_needs_*`?